x-pack/plugins/reporting/server/config/schema.test.ts

```diff
@@ -126,6 +126,34 @@ describe('Reporting Config Schema', () => {
     ).toMatchObject({ hostname: 'Frodo' });
   });
 
+  it('allows setting a wildcard for chrome proxy bypass', () => {
+    expect(
+      ConfigSchema.validate({
+        capture: {
+          browser: {
+            chromium: {
+              proxy: {
+                enabled: true,
+                server: 'http://example.com:8080',
+                bypass: ['foo.example.com', 'bar.example.com', 'bats.example.com'],
```

Comment:
This test checks for hostname literals - should it have a `*.example.com` instead?